egs2/Mustard/asr1/local/data.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
```

Comment:
It seems to be identical to ASVSPoof recipe (while only the dataset path is changed)?